src/java.base/linux/classes/jdk/internal/platform/cgroupv1/CgroupV1Subsystem.java

```diff
@@ -150,8 +150,9 @@ private static CgroupV1Subsystem initSubSystem(Map<String, CgroupInfo> infos) {
     }
 
     private static boolean getSwapEnabled(CgroupV1MemorySubSystemController controller) {
-         long retval = getLongValue(controller, "memory.memsw.limit_in_bytes");
-         return retval > 0;
+        long memswBytes = getLongValue(controller, "memory.memsw.limit_in_bytes");
+        long swappiness = getLongValue(controller, "memory.swappiness");
+        return (memswBytes > 0 && swappiness > 0);
```

Comment:
Does this also need to be changed in the test?

https://github.com/openjdk/jdk/blob/c63fabe3d582ce0828b04b0224cea49aab5fedf3/test/lib/jdk/test/lib/containers/cgroup/MetricsTesterCgroupV1.java#L291-L296

Reply:
There's also corresponding code in HotSpot:

https://github.com/openjdk/jdk/blob/c63fabe3d582ce0828b04b0224cea49aab5fedf3/src/hotspot/os/linux/cgroupV1Subsystem_linux.cpp#L129-L150

Reply:
> Does this also need to be changed in the test?
> 
> https://github.com/openjdk/jdk/blob/c63fabe3d582ce0828b04b0224cea49aab5fedf3/test/lib/jdk/test/lib/containers/cgroup/MetricsTesterCgroupV1.java#L291-L296


This condition returns false,The following code is skipped
https://github.com/openjdk/jdk/blob/c63fabe3d582ce0828b04b0224cea49aab5fedf3/test/lib/jdk/test/lib/containers/cgroup/MetricsTesterCgroupV1.java#L284